tide_core.module

```diff
@@ -58,6 +60,103 @@ function tide_core_entity_operation(EntityInterface $entity) {
       }
     }
   }
+  if (\Drupal::moduleHandler()->moduleExists('scheduled_transitions') && $entity->getEntityType()->hasLinkTemplate(ScheduledTransitionsRouteProvider::LINK_TEMPLATE_ADD)) {
```

Comment:
@vincent-gao we should not check for the module here as it is now a dependency. But we need to enable it in a hook_update of tide_core as the module is a new dependency.

Reply:
Fixed. thanks @sonnykt